src/CONST.ts

```diff
@@ -1741,6 +1743,7 @@ const CONST = {
     MAX_64BIT_MIDDLE_PART: 7203685,
     MAX_64BIT_RIGHT_PART: 4775807,
     INVALID_CATEGORY_NAME: '###',
+    INVALID_TAG_NAME: '###',
```

Comment:
Tags don't have this naming restriction.

Reply:
Handled.